cfgov/cfgov/urls.py

```diff
@@ -151,20 +151,10 @@ def empty_200_response(request, *args, **kwargs):
         template_name='fair-lending/index.html'),
         name='fair-lending'),
 
-    re_path(
-        r'^consumer-tools/educator-tools/students/knowbeforeyouowe/$',
-        TemplateView.as_view(
-            template_name='students/knowbeforeyouowe/index.html'),
-        name='students-knowbeforeyouowe'
-    ),
-    re_path(
-        r'^consumer-tools/educator-tools/students/'
-        'helping-borrowers-find-ways-to-stay-afloat/$',
-        TemplateView.as_view(
-            template_name='students/helping-borrowers-find-'
-            'ways-to-stay-afloat/index.html'),
-        name='students-helping-borrowers'
-    ),
+    # KBYO redirect
```

Comment:
Andy, I wasn't sure if I should be doing this, or messing around in https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/apache/conf.d/redirects.conf#L253 ???